ide/che-core-ide-generators/src/main/java/org/eclipse/che/util/GeneratorUtils.java

```diff
@@ -63,10 +67,34 @@ public static File getRootFolder(String[] args) {
                 rootDirPath = args[0].substring(ROOT_DIR_PARAMETER.length());
             } else {
                 System.err.print("Wrong usage. There is only one allowed argument : "
-                        + ROOT_DIR_PARAMETER);//NOSONAR
+                                 + ROOT_DIR_PARAMETER);//NOSONAR
                 System.exit(1);//NOSONAR
             }
         }
         return new File(rootDirPath);
     }
+
+    /**
+     * Parse command line arguments in format --key=value --key2=value2.
+     * Multiple keys is allowed
+     * @param args - command line argument.
```

Comment:
Doc has wrong formatting. Empty line before @param, new line after @param, `-` after @param is redundant
P.S. Use `ctrl`+`q` to preview generated java doc.